ubuntu/python/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM databricksruntime/minimal:latest
+
```

Comment:
Can we put a brief explanation in here why we are installing virtualenv and what these packages are for/doing?